src/web/src/app/app-routing.module.ts

```diff
@@ -9,17 +9,24 @@ import { spaceGuard } from './space/guard';
 import { LogoutComponent } from "./auth/logout/logout.component";
 import { LoginComponent } from "./auth/login/login.component";
 import { UserTestComponent } from "./user/user/user.component";
-import { HelloComponent } from "./space/hello/component";
+import { SpaceListComponent } from "./space/list/component";
+import { SpaceAddComponent } from "./space/add/component";
+import { SpaceDetailComponent } from "./space/detail/component";
+import { SpaceRequestSentComponent } from './space/space-request-sent/space-request-sent.component';
```

Comment:
Предлагаю файлы новых создаваемых компонентов называть просто component, так как имя компонента есть в названии папки. 
```suggestion
import { SpaceRequestSentComponent } from './space/space-request-sent/component';
```

Reply:
Готово.